### DISCUSSION TITLE
页面无法正常部署，出现乱码

### DISCUSSION THREAD

**Discussion by wz9523** [imsyy/home#96] [General]
页面无法正常部署无法进入动画，或者加载了动画”加载中“字体无法完全消失和其他文件无法被正常调用，想问一下大大这个问题怎么解决
![屏幕截图_20230220_000308](https://user-images.githubusercontent.com/120145864/219959794-57a74328-711f-4021-8e23-b6fc8448bd3d.png)

**Comment by imsyy** [comment]
不要直接打开 dist 文件夹下的 index.html，要么用 vscode 的 live server，要么上传到服务器并正确设置根目录后访问

**Comment by wz9523** [comment]
上传至静态资源服务器，该如何设置引导文件？

iPhone
________________________________
发件人: 底层用户 ***@***.***>
发送时间: Monday, February 20, 2023 8:57:52 AM
收件人: imsyy/home ***@***.***>
抄送: wz9523 ***@***.***>; Author ***@***.***>
主题: Re: [imsyy/home] 页面无法正常部署，出现乱码 (Issue #92)


不要直接打开 dist 文件夹下的 index.html，要么用 vscode 的 live server，要么上传到服务器并正确设置根目录后访问

―

**Comment by imsyy** [comment]
如果是宝塔，那么这样即可（home 是你 dist 文件夹下的内容上传到服务器上的文件夹的名称）
![image](https://user-images.githubusercontent.com/42232682/219993346-6e1a542e-d198-4581-9668-1b5031a8aacc.png)
其他服务器类似

**Comment by wz9523** [comment]
非常感谢您的指导，但是上传之后还是有这个问题，临时链接地址：https://www-5gnhholte9252807-1316678427.tcloudbaseapp.com

![屏幕截图_20230220_122800](https://user-images.githubusercontent.com/120145864/220009753-86152ed6-411a-49e4-82bb-ed7ab9b57c97.png)
![屏幕截图_20230220_122741](https://user-images.githubusercontent.com/120145864/220009761-8934fc7e-7d28-430f-bd08-5a3808c7c01f.png)
![屏幕截图_20230220_122810](https://user-images.githubusercontent.com/120145864/220009762-be727e2c-963c-420e-90e1-45c1e12e7b11.png)

**Comment by imsyy** [comment]
只上传dist文件夹下生成的内容

**Comment by wz9523** [comment]
已按照您的要求上传了dev-dist文件夹下的内容，但是依旧不可以正常访问，可能是我没有准确的理解您的意思，可以详细的指导一下吗。如果您方便的话我希望可以有您的联系方式能够更快的与您沟通。我也很抱歉打扰到了您。
![屏幕截图_20230220_130503](https://user-images.githubusercontent.com/120145864/220014472-821d0157-9575-41ef-aa62-a16c0692473d.png)

**Comment by imsyy** [comment]
不是 😂，是运行 `yarn dev` 后生成的 dist 文件夹下的文件
详细内容可以去 https://github.com/imsyy/home/discussions/61#discussioncomment-4840642 这里看看

**Comment by wz9523** [comment]
好的，非常感谢大大

iPhone
________________________________
发件人: 底层用户 ***@***.***>
发送时间: Monday, February 20, 2023 1:15:04 PM
收件人: imsyy/home ***@***.***>
抄送: wz9523 ***@***.***>; Author ***@***.***>
主题: Re: [imsyy/home] 页面无法正常部署，出现乱码 (Issue #92)


不是 😂，是运行 yarn dev 后生成的 dist 文件夹下的文件
详细内容可以去 #61 (comment)<https://github.com/imsyy/home/discussions/61#discussioncomment-4840642> 这里看看
